### ISSUE TITLE
Group message was send to individual chat.

### ISSUE THREAD

**Issue by kollokollo** [deltachat/deltachat-android#202]
I saw following strange behavvior:

1. I created a group with 4 members. To two of them I have also an individual chat.
2. After a while (weeks) of not using the group chat,  I deletet the group chat.
3. A message "xxx has left the group" was send to at least one former member of that group but it did not appear in the group chat, but instead in the individual chat with me.

Info: The group had never been used by anybody.

**Comment by r10s** [comment]
Are you sure, you have not at least send the "Hey, I have created a group message?".
If this is not send, the group has the internal state "In creation" and adding, removing and so on can be done without a group message being sent. At least this is the desired behaviour :)

**Comment by kollokollo** [comment]
Hm, yes, I have not let it send out the "Hey, I have created a group" message, because I did not want the members know, that I created it (because at this time I wasnt sure, If I wanted it at all). Ok, this might explain it. So the recipients never hat a chat with this group and the "xxx has left the group" message could not be associated with any group and such has been added to the existing individual chats (which caused confusion there). Maybe this is a rare case of unnormal use.

**Comment by r10s** [comment]
Well, but normally, the "xxx has left the group message" is not send if the "Hey, I have created a group"  was not sent before. 
(before the creation-message is sent, the groups has a special "unpromoted" state and status mails are not send, see eg. [here](https://github.com/deltachat/deltachat-core/blob/master/src/mrchat.c#L1888) for removing group members.

A little bit weird. Would be cool if we can reproduce this in some way.

BTW: 
![bildschirmfoto vom 2017-11-10 00-27-00](https://user-images.githubusercontent.com/9800740/32634934-56da6cdc-c5ae-11e7-997c-4234be18d024.png)
:)

**Comment by kollokollo** [comment]
I must admit, that since it was long ago, I do not really remember, if I have let it send the "Hey, I have created a group" message or not. But there are indications, that I havent: The one recipient, that I have looked at, did not have the group as a chat. So I assume, that the first message has not been sent. It really depends on a test for reproduction.  BTW: how did I not let it sent the greeting message: I just deleted the message with backspace until it was empty and then delta chat does not send anything.
